internals/cli/cmd_notice_test.go

```diff
@@ -115,6 +118,49 @@ func (s *PebbleSuite) TestNoticeTypeKey(c *C) {
 	c.Check(rest, HasLen, 0)
 	c.Check(s.Stdout(), Equals, `
 id: "123"
+user-id: 1000
```

Comment:
We should also add a test of the YAML marshalling for a public notice, ensuring that it comes through with `user-id: null` explicitly (I think that's what we want, rather than `omitempty`).

Reply:
Yes, definitely agree we want `user-id: null` rather than `omitempty`. I'll add a test for public notices too, thanks!